docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -94,15 +94,15 @@ head to `My Docker Runs <https://app.lightly.ai/docker/runs>`_
 
 .. image:: ../getting_started/images/docker_runs_overview.png
 
-Use your subsampled dataset
+Use your selected dataset
 ---------------------------
 
-Once the docker run has finished, you can see your subsampled dataset in the Lightly Platform:
+Once the docker run has finished, you can see your selected dataset in the Lightly Platform:
```

Comment:
see your selected dataset in -> see your dataset with the selected images in

Reply:
see https://github.com/lightly-ai/lightly/pull/735#discussion_r845343726